components/apimgt/org.wso2.carbon.apimgt.usage/org.wso2.carbon.apimgt.usage.client/src/main/java/org/wso2/carbon/apimgt/usage/client/impl/APIUsageStatisticsRestClientImpl.java

```diff
@@ -2498,6 +2522,6 @@ public int compare(Result<ThrottleDataOfAPIAndApplicationValue> o1, Result<Throt
      */
     @Override
     public String getClientType() {
-        return clientType;
+        return "REST";
```

Comment:
fixed in commit: https://github.com/wso2/carbon-apimgt/commit/8acbb9870e81a1141856fafee4b836d99bdbc74d